### PR TITLE
Rollback notes about certain pending to receivable changes for RPC co…

### PR DESCRIPTION
**Pull request by zhyatt**
…mmands

Related to https://github.com/nanocurrency/nano-node/issues/3689

### CODE REVIEW

File: docs/releases/release-v23-0.md [hunk 1]
```diff
@@ -56,25 +56,20 @@ Updates to simplify the unit names used within the node wallet and unit conversi
...
-* [`search_pending`](../commands/rpc-protocol.md#search_pending) replaced by [`search_receivable`](../commands/rpc-protocol.md#search_receivable)
-* [`search_pending_all`](../commands/rpc-protocol.md#search_pending_all) replaced by [`search_receivable_all`](../commands/rpc-protocol.md#search_receivable_all)
-* [`wallet_pending`](../commands/rpc-protocol.md#wallet_pending) replaced by [`wallet_receivable`](../commands/rpc-protocol.md#wallet_receivable)
 
 **Response/request key changes only**
 
 * [`account_balance`](../commands/rpc-protocol.md#account_balance)
 * [`account_info`](../commands/rpc-protocol.md#account_info)
 * [`accounts_balances`](../commands/rpc-protocol.md#accounts_balances)
-* [`blocks_info`](../commands/rpc-protocol.md#blocks_info)
```
Comment (thsfs) [inline docs/releases/release-v23-0.md:77]: This doesn't seem to be related to the pending/receivable change.

Comment (zhyatt) [reply docs/releases/release-v23-0.md]: See optional parameters on both account_info and blocks_info for how these are related. I realized that only part of the desired solution is implemented on this (response contains `receivable` when optional parameter `pending` is given) so it probably should be left in as a reference. RPC commands page reflects the proper behavior now and the related node repository ticket has been updated to ensure the right changes are made in the future.

**Review by thsfs — APPROVED** [review]
